tests/test-sources/modules/lua-loader.nix

```diff
@@ -7,11 +7,11 @@
 
       assertions = [
         {
-          assertion = builtins.match ".*vim\.loader.*" config.content == null;
+          assertion = builtins.match ".*vim\\.loader.*" config.content == null;
```

Comment:
We could also do `[.]`, but if this works on all platforms then that's fine.

I'm curious, is `"\\."` the same as `''\.''`? If so maybe using raw strings for regexes would be better?

Reply:
Actually, we can replace all these with `lib.hasInfix`, which just checks if a string contains a given substring.

It's implemented using `match` and a regex escape, but is better than rolling our own regexes.

Reply:
> Actually, we can replace all these with `lib.hasInfix`, which just checks if a string contains a given substring.
> 
> It's implemented using `match` and a regex escape, but is better than rolling our own regexes.

100%, I’d prefer that to dealing with a regex ourselves. I’ll update in a bit